doc/1.10/dev_guide/building_from_source.rst

```diff
@@ -186,6 +190,11 @@ the preferences can differ. But strategically the steps are always the same.
 
    This creates the 'tarantool' executable in the ``src/`` directory.
 
+   .. NOTE::
+
+       If you encounter a ``curl`` or ``OpenSSL`` errors on this step try
```

Comment:
If you encounter ``curl`` or ``OpenSSL`` errors on this step, try
installing an ``openssl111`` package of the specific 1.1.1d version.